client/src/components/StreamField/blocks/ListBlock.js

```diff
@@ -130,12 +130,13 @@ export class ListBlock extends BaseSequenceBlock {
     }
     const error = errorList[0];
 
-    // eslint-disable-next-line no-restricted-syntax
-    for (const blockIndex in error.blockErrors) {
-      if (error.blockErrors.hasOwnProperty(blockIndex)) {
-        this.children[blockIndex].setError(error.blockErrors[blockIndex]);
+    // error.blockErrors = a list with the same length as the data,
+    // with nulls for items without errors
+    error.blockErrors.forEach((blockError, blockIndex) => {
```

Comment:
Does the forEach here work on IE11?

Reply:
We're using forEach extensively elsewhere in the StreamField implementation, so if that's causing problems with IE11, that ship has sailed by now :-) Doesn't babel / core-js deal with that, though? (Our webpack.config.js adds `client/src/utils/polyfills.js` to all bundles, which in turn imports `core-js/shim`.)